src/grubconfigs.rs

```diff
@@ -25,6 +26,7 @@ const GRUBCONFIG_FILE_MODE: u32 = 0o600;
 #[context("Installing static GRUB configs")]
 pub(crate) fn install(
     target_root: &openat::Dir,
+    src_root: Option<&openat::Dir>,
```

Comment:
Yeah one problem we keep getting bit by here is that the openat crate is old and unmaintained, I'd like to move this project over to cap-std which we use *extensively* in bootc at least.